packages/search/src/util/aggregations/merge-aggregations.ts

```diff
@@ -0,0 +1,159 @@
+/**
+ * Interface for an Aggregation returned from search request,
+ * where label is a value for a fieldname type for which aggregations were requested
+ * and aggValue is the aggregation value
+ */
+interface IAggregation {
+  label: string;
+  aggValue: number;
```

Comment:
suggest `value`. To me `aggregation.value` is better than `aggregation.aggValue`.

`

Reply:
Thanks I have fixed. That makes more sense.